### PR TITLE
Release Notes: TinyMCE 5.0.13

### PR DESCRIPTION
**Pull request by tylerkelly13**
Adding Release notes for the 5.0.13 TinyMCE release.

### CODE REVIEW

File: release-notes/release-notes5013.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+content on how to use the editor when navigating via a keyboard.
+
+#### Nonbreaking space plugin
+
+TinyMCE 5.0.13 improves the nonbreaking space plugin by wrapping any nonbreaking space characters inserted by the plugin in a span to prevent the character being converted to a regular space while typing. To disable this behavior, add the new [`nonbreaking_wrap: false`]({{site.baseurl}}/plugins/nonbreaking/#nonbreaking_wrap) setting to the TinyMCE init configuration.
+
+#### Search and replace plugin
+
+The search and replace dialog will now appear inline, instead of as a modal dialog, preventing it from overlapping the content found.
+The dialog also comes with a new design that's better suited to an inline dialog.
```
Comment (blackcathikari) [inline release-notes/release-notes5013.md]: needs an extra newline? XD

File: release-notes/release-notes5013.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+### TinyMCE
+
+The updated TinyMCE 5.0.13 editor comes with numerous changes to the previous versions. A comprehensive list of the updates
+and fixed issues is available in the [changelog]({{site.baseurl}}/changelog/#version5013august62019).
+
+#### Accessibility improvements
+
+This release provides accessibility and keyboard navigation fixes for inline dialogs. Users will now be able to tab through the input and button elements in
+the dialog without losing cursor focus when closing the preferences menu.
```
Comment (blackcathikari) [inline release-notes/release-notes5013.md]: Actually, what do you mean by the "preferences menu"? That sounds like it's specific to the searchreplace dialog...

Comment (lnewson) [reply release-notes/release-notes5013.md]: Ahh good catch, in my notes I copied this from it was specifically about the search and replace dialog. We should definitely make this more generic.

Comment (lnewson) [reply release-notes/release-notes5013.md]: Maybe something like this?
```
This release provides accessibility and keyboard navigation fixes for inline dialogs. Users will now be able to tab through the input and button elements in the dialog without the cursor getting stuck or losing focus.
```

File: release-notes/release-notes5013.md (hunk 1 above)
Comment (tylerkelly13) [reply release-notes/release-notes5013.md]: updated by 400e59fdb2a7173667ec01a6572496e75139ebe5

File: release-notes/release-notes5013.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes5013.md]: updated by 400e59fdb2a7173667ec01a6572496e75139ebe5

**Pull request merged**